multidict/_multidict.c

```diff
@@ -863,6 +864,22 @@ multidict_class_getitem(PyObject *self, PyObject *arg)
     return self;
 }
 
+
+PyDoc_STRVAR(sizeof__doc__,
+"D.__sizeof__() -> size of D in memory, in bytes");
+
+static inline PyObject *
```

Comment:
`inline` is not necessary

Reply:
Yes, I know.
I've converted all functions to inlines after adding `_multilib`, let's keep this style at least for a while. It doesn't hurt.